subsys/bluetooth/controller/ll_sw/ull_conn_types.h

```diff
@@ -220,14 +195,42 @@ struct ll_conn {
 #endif /* CONFIG_BT_CENTRAL */
 	};
 
+	struct ull_tx_q tx_q;
+	struct llcp_struct llcp;
+
+	/* ULL tracked connection event counter. Under LLL Prepare deferred cases this member
+	 * accumulates the connection event count, necessary for LLCP instant use in ULL.
+	 */
+	uint16_t event_counter;
+
 	/* Cancel the prepare in the instant a Connection Update takes place */
 	uint8_t cancel_prepare:1;
 
+	/*
+	 * TODO: all the following comes from the legacy LL llcp structure
+	 * and/or needs to be properly integrated in the control procedures
+	 */
```

Comment:
This sounds like something we should have a GH issue for - Sounds like a fairly large thing